tests/Makefile

```diff
@@ -0,0 +1,32 @@
+# Copyright (c) 2024, NVIDIA CORPORATION.  All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+GO_CMD ?= go
+
+E2E_ENV_FILE ?= $(CURDIR)/tests/test_env.yml
+LOG_ARTIFACTS ?= $(CURDIR)/e2e_logs
+
+.PHONY: e2e-test
+e2e-test:
+	@if [ -z ${AWS_SECRET_ACCESS_KEY} ] || [ -z ${AWS_ACCESS_KEY_ID} ]; then \
```

Comment:
Question: Doesn't holodeck already report an error if these are not set?

Reply:
the aws-sdk pkg will complain about it, but we don't have a proper check before calling the aws-sdk pkg's.

Reply:
My suggestion would be to remove this here (for simplicity) and create an issue to check these in code so that that can be reused in all consumers (device plugin, gpu operator, container toolkit) etc.